templates/core/about/builds.html

```diff
@@ -31,12 +31,10 @@ <h4 id="setting-a-readme"> <a href="#setting-a-readme">Setting a README</a> </h4
     <h4 id="detecting-docsrs"> <a href="#detecting-docsrs">Detecting Docs.rs</a> </h4>
     <p>
         To recognize Docs.rs from <code>build.rs</code> files, you can test for the environment variable <code>DOCS_RS</code>, e.g.:
-        {% filter dedent(levels=4) -%}
-        <pre><code class="lang-rust">
-            if let Ok(_) = std::env::var("DOCS_RS") {
+        {% filter dedent(levels=3) -%}
```

Comment:
Hopefully the removal of the newline after `<pre><code>` will interact nicely with this dedent level...